tinyquery/evaluator.py

```diff
@@ -157,9 +165,15 @@ def evaluate_orderings(self, overall_context, select_context,
             all_values.append(column.values)
 
         for order_by_column in ordering_col:
-            for count, ((_, column_name), column) in enumerate(
+            order_column_name = order_by_column.column_id.name
+
+            for count, (column_identifier, column) in enumerate(
                     overall_context.columns.iteritems()):
-                if order_by_column.column_id.name == column_name:
+                if (
+                    '%s.%s' % column_identifier == order_column_name
```

Comment:
Can you call this `column_identifier_pair` or something else to make it clearer it's a tuple?  Otherwise this string interpolation is really confusing.